gulpfile.babel.js

```diff
@@ -1,334 +1,46 @@
-		.on( 'end', function() {
-			log( 'dops-components CSS finished.' );
-			doRTL( 'dops', done );
-		} );
-}
-
-function doRTL( files, done ) {
-	gulp.src( 'main' === files ? './_inc/build/style.min.css' : './_inc/build/*dops-style.css' )
-		.pipe( rtlcss() )
-		.pipe( rename( { suffix: '.rtl' } ) )
-		.pipe( sourcemaps.init() )
-		.pipe( sourcemaps.write( './' ) )
-		.pipe( gulp.dest( './_inc/build' ) )
-		.on( 'end', function() {
-			log( 'main' === files ? 'Dashboard RTL CSS finished.' : 'DOPS Components RTL CSS finished.' );
-			if ( done && 'function' === typeof done ) {
-				done();
-			}
-		} );
-}
-
-gulp.task( 'sass:build', [ 'react:build' ], doSass );
-
-gulp.task( 'sass:watch', function() {
-	doSass();
-	gulp.watch( [ './**/*.scss', ...alwaysIgnoredPaths ], doSass );
-} );
-
-gulp.task( 'react:build', function( done ) {
-	const config = getWebpackConfig();
-
-	if ( 'production' !== process.env.NODE_ENV ) {
-		config.plugins.push(
-			new webpack.LoaderOptionsPlugin( {
-				debug: true
-			} )
-		);
-	}
+import { alwaysIgnoredPaths } from './tools/builder/util';
 
-	webpack( config ).run( onBuild( done ) );
-} );
-
-gulp.task( 'react:watch', function() {
-	const config = getWebpackConfig();
-
-	webpack( config ).watch( 100, onBuild() );
-} );
-
-function doStatic( done ) {
-	let path;
-
-	const jsdom = require( 'jsdom' );
-
-	log( 'Building static HTML from built JS…' );
-
-	jsdom.env( '', function( err, window ) {
-		global.window = window;
-		global.document = window.document;
-		global.navigator = window.navigator;
-
-		window.Initial_State = {
-			dismissedNotices: [],
-			connectionStatus: {
-				devMode: {
-					isActive: false
-				}
-			},
-			userData: {
-				currentUser: {
-					permissions: {}
-				}
-			}
-		};
-
-		try {
-			path = __dirname + '/_inc/build/static.js';
-
-			delete require.cache[ path ]; // Making sure NodeJS requires this file every time this is called
-			require( path );
-
-			gulp.src( [ '_inc/build/static*' ] )
-				.pipe( tap( function( file ) {
-					fs.unlinkSync( file.path );
-				} ) )
-				.on( 'end', function() {
-					fs.writeFileSync( __dirname + '/_inc/build/static.html', window.staticHtml );
-					fs.writeFileSync( __dirname + '/_inc/build/static-noscript-notice.html', window.noscriptNotice );
-					fs.writeFileSync( __dirname + '/_inc/build/static-version-notice.html', window.versionNotice );
-					fs.writeFileSync( __dirname + '/_inc/build/static-ie-notice.html', window.ieNotice );
-
-					if ( done ) {
-						done();
-					}
-				} );
-		} catch ( error ) {
-			log( colors.yellow(
-				'Warning: gulp was unable to update static HTML files.\n\n' +
-				'If this is happening during watch, this warning is OK to dismiss: sometimes webpack fires watch handlers when source code is not yet built.'
-			) );
-		}
-	} );
-}
+import frontendcss from './tools/builder/frontend-css';
+import admincss from './tools/builder/admin-css';
+import {
+    watch as react_watch,
+    build as react_build
```

Comment:
Nitpick -- let's avoid `snake_case` and use `camelCase` instead (same for other imports/aliases).